### ISSUE TITLE
[Bug] type: lines, select is invalid

### ISSUE THREAD

**Issue by DoubleCorner** [apache/echarts#16981]
### Version

5.3.2

### Steps to Reproduce

1. https://echarts.apache.org/examples/zh/editor.html?c=lines-bmap 官网的 demo 就可以复现
2. 增加如下配置：
3. `select: {
          lineStyle: {
            color: 'blue'
          }
        },
        selectedMode: true
`
4. 点击路径后，颜色不会被设置为蓝色

### Current Behavior

配置之后，select 无效

### Expected Behavior

select 配置后，点击线条，颜色可以变为蓝色

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
yes, both [lines.select.lineStyle](https://echarts.apache.org/en/option.html#series-lines.select.lineStyle) and [line.select.lineStyle](https://echarts.apache.org/en/option.html#series-line.select.lineStyle) are non-functional.
